[2025-09-05T22:59:01.719+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-05T22:59:01.749+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-05T22:58:00+00:00 [queued]>
[2025-09-05T22:59:01.758+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-05T22:58:00+00:00 [queued]>
[2025-09-05T22:59:01.759+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 11
[2025-09-05T22:59:01.772+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): save_foreign_1> on 2025-09-05 22:58:00+00:00
[2025-09-05T22:59:01.781+0000] {standard_task_runner.py:63} INFO - Started process 80275 to run task
[2025-09-05T22:59:01.786+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'foreign_1_live', 'save_foreign_1', 'scheduled__2025-09-05T22:58:00+00:00', '--job-id', '122128', '--raw', '--subdir', 'DAGS_FOLDER/dag_foreign_1_live.py', '--cfg-path', '/tmp/tmpfjx4i3_z']
[2025-09-05T22:59:01.789+0000] {standard_task_runner.py:91} INFO - Job 122128: Subtask save_foreign_1
[2025-09-05T22:59:01.843+0000] {task_command.py:426} INFO - Running <TaskInstance: foreign_1_live.save_foreign_1 scheduled__2025-09-05T22:58:00+00:00 [running]> on host d4d2fe648a81
[2025-09-05T22:59:01.946+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='foreign_1_live' AIRFLOW_CTX_TASK_ID='save_foreign_1' AIRFLOW_CTX_EXECUTION_DATE='2025-09-05T22:58:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-09-05T22:58:00+00:00'
[2025-09-05T22:59:01.947+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-05T22:59:01.961+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-05T22:59:02.517+0000] {foreign_1.py:28} INFO - Đã lưu foreign_ALL
[2025-09-05T22:59:02.518+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-05T22:59:03.048+0000] {foreign_1.py:28} INFO - Đã lưu foreign_HOSE
[2025-09-05T22:59:03.049+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-05T22:59:03.598+0000] {foreign_1.py:28} INFO - Đã lưu foreign_HNX
[2025-09-05T22:59:03.600+0000] {foreign_1.py:18} INFO - Kết nối DB
[2025-09-05T22:59:04.183+0000] {foreign_1.py:28} INFO - Đã lưu foreign_UPCOM
[2025-09-05T22:59:04.184+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-09-05T22:59:04.185+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-05T22:59:04.204+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=foreign_1_live, task_id=save_foreign_1, run_id=scheduled__2025-09-05T22:58:00+00:00, execution_date=20250905T225800, start_date=20250905T225901, end_date=20250905T225904
[2025-09-05T22:59:04.244+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-09-05T22:59:04.285+0000] {taskinstance.py:3498} INFO - 4 downstream tasks scheduled from follow-on schedule check
[2025-09-05T22:59:04.286+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
